presto-docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -76,12 +76,22 @@ For example, setting ``region,rack,machine`` means a network location contains t
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * **Type:** ``string``
-* **Allowed values:** ``flat``, ``file``
+* **Allowed values:** ``flat``, ``file``, ``subnet``
 * **Default value:** ``flat``
 
-Sets the network topology type. To use this option, ``node-scheduler.policy`` must be set to
-``topology``. ``flat`` has only one segment, with one value for each machine.
-``file`` loads the topology from a file as described below.
+Sets the network topology type. To use this option, ``node-scheduler.policy``
+must be set to ``topology``.
+
+``flat`` : the topology has only one segment, with one value for each machine.
+
+``file``: the topology is loaded from a file using
+``node-scheduler.network-topology.file`` and
+``node-scheduler.network-topology.refresh-period`` properties described below.
+
+``subnet``: the topology is derived based on subnet configuration provided
+through properties ``node-scheduler.network-topology.subnet.cidr-prefix-lengths``
+and ``node-scheduler.network-topology.subnet.ip-address-protocol`` described
+below.
```

Comment:
described in the following sections.

Reply:
also .. from a file using the ...

Reply:
"from a file" doesn't seem appropriate here as we expect these configurations to be in `config.properties`. did you mean adding a "the" for the sentence in  ``file`` section?

Reply:
yeah .. sorry .. it was above this section